modules/ivc_champva/spec/requests/ivc_champva/v1/forms/ves_uploads_spec.rb

```diff
@@ -28,136 +29,144 @@
     # Mock file uploads
     allow(PersistentAttachments::MilitaryRecords).to receive(:find_by)
       .and_return(double('Record1', created_at: 1.day.ago, id: 'some_uuid', file: double(id: 'file0')))
-    allow_any_instance_of(Aws::S3::Client).to receive(:put_object).and_return(
-      double('response', context: double('context', http_response: double('http_response', status_code: 200)))
-    )
+    allow(IvcChampva::S3).to receive(:new).and_return(mock_s3)
+    allow(mock_s3).to receive(:put_object).and_return({ success: true })
```

Comment:
Reworked how `put_object` is being mocked because it was resulting in flaky behavior.  I'm not sure now it was getting proper values some of the time since before this change `success` was not being set.  My best guess is that previously this was mocking `Aws::S3::Client` not `IvcChampva::S3` and values for `IvcChampva::S3` were leaking in somehow.